packages/web-react/src/components/Toast/useToastBarStyleProps.ts

```diff
@@ -7,20 +7,24 @@ export const useToastBarStyleProps = (props: ToastBarProps) => {
 
   const toastBarClass = useClassNamePrefix('ToastBar');
   const toastBarBoxClass = `${toastBarClass}__box`;
+  const toastBarCloseClass = `${toastBarClass}__close`;
   const toastBarContainerClass = `${toastBarClass}__container`;
   const toastBarContentClass = `${toastBarClass}__content`;
   const toastBarLinkClass = `${toastBarClass}__link`;
-  const colorClass = `${toastBarClass}--${color || 'inverted'}`;
+  const toastBarLinkUnderlinedClass = useClassNamePrefix('link-underlined');
+  const colorClass = `${toastBarClass}--${color || 'neutral'}`;
   const dismissibleClass = `${toastBarClass}--dismissible`;
   const rootClass = classNames(toastBarClass, colorClass, isDismissible && dismissibleClass);
+  const linkClass = classNames(toastBarLinkClass, toastBarLinkUnderlinedClass);
```

Comment:
It doesn't matter much but I think all constants here should either start with `toastBar`.